llumnix/arg_utils.py

```diff
@@ -16,12 +16,11 @@
 import argparse
 from typing import Tuple
 
-from llumnix.config import GlobalSchedulerConfig, MigrationConfig
-
+from llumnix.llumnix_config import GlobalSchedulerConfig, MigrationConfig
```

Comment:
To echo my previous comment on naming:
```
from llumnix.llumnix_config import GlobalSchedulerConfig, MigrationConfig
from llumnix.config.config import LlumnixConfig
```

This code is pretty confusing. Please refine these names